packages/solid/src/server/rendering.ts

```diff
@@ -552,7 +552,6 @@ export function SuspenseList(props: {
 
 export function Suspense(props: { fallback?: string; children: string }) {
   let done: undefined | ((html?: string, error?: any) => boolean);
-  let clean: any;
```

Comment:
`clean` wasn't being used anywhere anymore. maybe that's a mistake? not sure what it did